source/randLIB.c

```diff
@@ -95,14 +95,13 @@ void randLIB_seed_random(void)
      * multiple times,or in case someone has called randLIB_add_seed before
      * this. We don't want to potentially lose entropy.
      */
-
-    srand(time(0));
+    randLIB_add_seed(time(NULL));
```

Comment:
Do you really need this? The comment on the last PR said that you believed apps/porting layers were always calling `randLIB_add_seed` themselves.

If that's NOT true, then these changes end up having non-randomised nodes, and no error report.

Many real embedded devices will not have their system clock set when this is called, and devices being booted en-mass will get the same value.

I would much rather have this function do nothing at all, leaving state as 0, and have an assert in `randLIB_get_64bit`: `assert(state[0] != 0 || state[1] != 0);`.

Although I guess we don't actually have a portable assert in libservice, do we? It's a similar dependency we don't currently have to `time`. Hmm.

Even just not having the assert, it would return a constant 0, due to the maths, the same as if `random_file` hadn't been initialised properly.  That might be a louder clearer failure?